index.html

```diff
@@ -483,15 +485,16 @@
           alphanumeric ASCII characters only, is at least 16 characters long,
           and is unique within the <a>set of presentations</a>.
         </p>
-        <p link-for="Presentation">
+        <p>
           A <dfn lt=
           "controlling browsing context|controlling browsing contexts">controlling
           browsing context</dfn> (or <dfn>controller</dfn> for short) is a
           <a>browsing context</a> that has initiated or resumed a
-          <a>presentation session</a> by calling
-          <code><a>startSession</a>()</code> or
-          <code><a>joinSession</a>()</code> or received a <a>presentation
-          session</a> via <code><a>ondefaultsessionstart</a></code> event.
+          <a>presentation session</a> by calling <code><a for=
+          "PresentationRequest">start</a>()</code> or <code><a for=
+          "PresentationRequest">join</a>()</code> or received a <a>presentation
+          session</a> via <code><a for=
+          "Presentation">defaultRequest</a></code>.
```

Comment:
I would use two sentences here, one for the page-initiated case and the other for the UA-initiated case.